src/UI/Component/Panel/Listing/Divider.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/* Copyright (c) 2017 Alex Killing <killing@leifos.de> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Component\Panel\Listing;
+
+/**
+ * Listing divider
+ * @package ILIAS\UI\Component\Panel\Listing
+ */
+interface Divider extends \ILIAS\UI\Component\Component {
```

Comment:
To sum it up.

IMHO: 
Groups have to advantage:
- They can do things like wrap their contained items in their own renderer and so enable thing like accordions in lists.
- The structure seems cleaner since it is clear what belongs to what.

Dividers have to advantage:
- They need less nesting and are so more comfortable to initiate
- They also allow to wrap items in divs but only through the parent container (e.g. Appointment Listing) and so also allow things like accordions.

Since this wrapping would happen in dividers in the rendering of listing themselves and in the groups in the renderer there, I think dividers are more strait forward. After all, if you would like to have different appointment listings (e.g. with accordions) but not different repository listings (without accordions) you would need special groups for appointments, right? With dividers you would just need two different listings, since the wrapping happens there.

At the end of the day, both are possible and have their trade-offs. Nevertheless we need to reach a decision how to proceed with this for 5.3. I have a preference for the divider. To be honest, also because @alex40724 provides at least parts of a solution for that already.

I see, that groups seem to be somehow the obvious concept here, but I do not like the additional complexity layer here. @klees Can you live with that?